lib/prometheus.sh

```diff
@@ -106,3 +113,35 @@ _load_prometheus_operator_images_to_local_registry() {
 		done
 	done
 }
+
+_setup_dashboard(){
+	if [ -f "$DASHBOARD_DIR/grafana-dashboards/kepler-exporter-configmap.yaml" ]; then
+		return 0
+	else
+	header "Setup Dashboard"
+	mkdir -p "$DASHBOARD_DIR/grafana-dashboards/"
+	cat - > "$DASHBOARD_DIR/grafana-dashboards/kepler-exporter-configmap.yaml" << EOF
+apiVersion: v1
+data:
+    kepler-exporter.json: |-
+        $KEPLER_EXPORTER_GRAFANA_DASHBOARD_JSON
+kind: ConfigMap
+metadata:
+    labels:
+        app.kubernetes.io/component: grafana
+        app.kubernetes.io/name: grafana
+        app.kubernetes.io/part-of: kube-prometheus
+        app.kubernetes.io/version: 9.5.3
+    name: grafana-dashboard-kepler-exporter
+    namespace: monitoring
+EOF
+    fi
+}
+
+_run_yq(){
```

Comment:
this should moved to verify shell to avoid yq command not found in CI, so far the dashboard seems not necessary for CI to verify.

Reply:
yq is updating the YAML files in place to add the grafana dashboard. I'm not sure this is a verify step TBH.

I can move it directly before the grafana deploy - no issue